### PR TITLE
Multisite 12962

### PR DESCRIPTION
**Pull request by GitDiaz**

### CODE REVIEW

File: lib/features/custom/erasmus_core/erasmus_core.install [hunk 1]
```diff
@@ -25,17 +25,31 @@ function erasmus_core_enable() {
 }
 
 /**
- * Implements hook_install().
+ * Erasmus core feature installed - hook_install().
```
Comment (verbruggenalex) [inline lib/features/custom/erasmus_core/erasmus_core.install]: Needs to stay "Implements hook_install()."

File: lib/features/custom/erasmus_core/erasmus_core.install [hunk 2]
```diff
@@ -15,7 +15,7 @@ function erasmus_core_disable() {
 }
 
 /**
- * Implements hook_enable().
+ * Erasmus core feature active -  hook_enable().
```
Comment (verbruggenalex) [inline lib/features/custom/erasmus_core/erasmus_core.install]: Needs to stay "Implements hook_enable()."

File: lib/features/custom/erasmus_core/erasmus_core.install [hunk 3]
```diff
@@ -25,17 +25,31 @@ function erasmus_core_enable() {
...
- * Implements hook_uninstall().
+ * Erasmus core feature unistalled - hook_uninstall().
  */
 function erasmus_core_uninstall() {
   $t = get_t();
   drupal_set_message($t('erasmus core feature is now uninstalled from your site.'));
 }
+
+/**
+ * Enable modules Simplify, Footnotes and Workbench - hook_update_N().
```
Comment (verbruggenalex) [inline lib/features/custom/erasmus_core/erasmus_core.install]: Remove the "- hook_update_N()" part

File: lib/features/custom/erasmus_core/erasmus_core.install [hunk 4]
```diff
@@ -25,17 +25,31 @@ function erasmus_core_enable() {
...
+ */
+function erasmus_core_update_7000() {
```
Comment (verbruggenalex) [inline lib/features/custom/erasmus_core/erasmus_core.install]: The number 7000 is reserved for upgrading from D6 to D7. Please read up on the way hook updates are run: https://api.drupal.org/api/drupal/modules!system!system.api.php/function/hook_update_N/7.x

**Comment by GitDiaz** [comment]
Done. Thanks.

**Pull request merged**